recipe/conda_build_config.yaml

```diff
@@ -6,3 +6,11 @@ cxx_compiler_version:      # [linux]
   - 10                     # [linux]
 fortran_compiler_version:  # [linux]
   - 10                     # [linux]
+
+channel_sources:
+- isuruf/label/m2,isuruf/label/ucrt64,conda-forge
+channel_targets:
+- conda-forge main
+# TODO: unpin again
+cairo:  # [win]
+- 1.16  # [win]
```

Comment:
I don't have a Windows VM up and running, so can't look into this now.
Excerpt of the error log:
```
2023-11-02T09:55:21.9430334Z gcc  -I"../../../../../include" -DNDEBUG -I../../../../../src/include -I. -I.. -DHAVE_CONFIG_H -DR_ARCH='"x64"' -ID:/bld/r-base-split_1698917465091/_h_env/Library//include/cairo -ID:/bld/r-base-split_1698917465091/_h_env/Library//include/freetype2    -I"D:/bld/r-base-split_1698917465091/_h_env/Library/include"     -O2 -Wall  -march=x86-64 -mtune=generic  -c cairoBM.c -o cairoBM.o
2023-11-02T09:55:23.2303891Z gcc  -shared -s -o winCairo.dll winCairo.def cairoBM.o -lcairo -lfontconfig -L.. -lgrDevices -L"D:/bld/r-base-split_1698917465091/_h_env/Library/lib/x64" -L"D:/bld/r-base-split_1698917465091/_h_env/Library/lib"  -L"../../../../../bin/x64" -lR 
2023-11-02T09:55:23.4721327Z Warning: corrupt .drectve at end of def file
<<snip>>
2023-11-02T09:55:23.4947740Z Warning: corrupt .drectve at end of def file
2023-11-02T09:55:23.5517561Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0x91): undefined reference to `__imp_cairo_push_group'
2023-11-02T09:55:23.5522288Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0xaa): undefined reference to `__imp_cairo_get_operator'
2023-11-02T09:55:23.5526138Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0xc4): undefined reference to `__imp_cairo_push_group'
2023-11-02T09:55:23.5529685Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0x1bd): undefined reference to `__imp_cairo_set_source_rgba'
2023-11-02T09:55:23.5533268Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0x1e9): undefined reference to `__imp_cairo_set_source_rgb'
2023-11-02T09:55:23.5537094Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0x21f): undefined reference to `__imp_cairo_pattern_destroy'
2023-11-02T09:55:23.5540653Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0x2fd): undefined reference to `__imp_cairo_pattern_destroy'
2023-11-02T09:55:23.5544208Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0x39f): undefined reference to `__imp_cairo_pattern_destroy'
2023-11-02T09:55:23.5547770Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0x47d): undefined reference to `__imp_cairo_pattern_destroy'
2023-11-02T09:55:23.5551322Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0x518): undefined reference to `__imp_cairo_pattern_destroy'
2023-11-02T09:55:23.5555724Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0x584): more undefined references to `__imp_cairo_pattern_destroy' follow
2023-11-02T09:55:23.5561527Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0xd51): undefined reference to `__imp_cairo_set_line_width'
2023-11-02T09:55:23.5568833Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0xd5c): undefined reference to `__imp_cairo_set_line_cap'
2023-11-02T09:55:23.5572199Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0xd68): undefined reference to `__imp_cairo_set_line_join'
2023-11-02T09:55:23.5575335Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0xd76): undefined reference to `__imp_cairo_set_miter_limit'
2023-11-02T09:55:23.5578390Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0xde4): undefined reference to `__imp_cairo_set_dash'
2023-11-02T09:55:23.5581414Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0xdff): undefined reference to `__imp_cairo_stroke'
2023-11-02T09:55:23.5584410Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0xe16): undefined reference to `__imp_cairo_set_dash'
2023-11-02T09:55:23.5587447Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0xf3f): undefined reference to `__imp_cairo_path_destroy'
2023-11-02T09:55:23.5591694Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0x101d): undefined reference to `__imp_cairo_path_destroy'
2023-11-02T09:55:23.5597056Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0x1099): undefined reference to `__imp_cairo_image_surface_get_data'
2023-11-02T09:55:23.5601147Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0x10b2): undefined reference to `__imp_cairo_image_surface_get_stride'
2023-11-02T09:55:23.5605191Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cairoBM.o:cairoBM.c:(.text+0x1106): undefined reference to `snprintf'
<<snip>>
2023-11-02T09:55:23.6259376Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/bld/r-base-split_1698917465091/_h_env/Library/lib/libcairo.a(src/cairo.dll.p/win32_cairo-dwrite-font.cpp.obj):(.text$mn+0x22): undefined reference to `?Instance@DWriteFactory@@SA?AV?$RefPtr@UIDWriteFactory@@@@XZ'
2023-11-02T09:55:23.6264062Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/bld/r-base-split_1698917465091/_h_env/Library/lib/libcairo.a(src/cairo.dll.p/win32_cairo-dwrite-font.cpp.obj):(.text$mn+0x84): undefined reference to `??_U@YAPEAX_K@Z'
<<snip>>
2023-11-02T09:55:24.5645287Z D:/bld/r-base-split_1698917465091/_h_env/Library/mingw-w64/bin/../lib/gcc/x86_64-w64-mingw32/13.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: D:/bld/r-base-split_1698917465091/_h_env/Library/lib/libcairo.a(src/cairo.dll.p/cairo-mesh-pattern-rasterizer.c.obj):(.xdata[$unwind$draw_bezier_patch]+0x14): undefined reference to `__GSHandlerCheck'
2023-11-02T09:55:24.5649339Z collect2.exe: error: ld returned 1 exit status
```